releasenotes/notes/kubelet-core-volume-metrics-fix-a95e22dbc29dea88.yaml

```
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fixes issue with the kubelet corecheck where `kubernetes.kubelet.volume.*` metrics
```

```suggestion
    Fixes issue with the kubelet core check where `kubernetes.kubelet.volume.*` metrics
```
Please ignore if "corecheck" is standard, I just haven't seen that before.